chain/jsonrpc/client/src/lib.rs

```diff
@@ -105,6 +106,7 @@ macro_rules! expand_params {
 
 /// Generates a simple HTTP client with automatic serialization and deserialization.
 /// Method calls get correct types automatically.
+/// cspell:ignore selff
```

Comment:
maybe "selff" is a typo

Reply:
I think we use selff as `self` is a rust keyword and can't be used in macros.